package/com.unity.formats.usd/Runtime/Scripts/IO/Materials/MaterialImporter.cs

```diff
@@ -155,6 +155,7 @@ System.Collections.Generic.List<string> primvars
         matAdapter.ImportFromUsd();
       }
 
+      mat.name = materialPath;
```

Comment:
I think I'd go with the material name instead of the full path. I'm not sure there's any value in having the full path as material name. And paths can get extremely long in the case of nested assets.

Reply:
will fix. I went with the path in case there would be an issue with multiple materials of the same name being declared at different paths.

Reply:
Yeah that's what I guessed. As Unity can deal with duplicated names on GO  I think we can live with that.